### ISSUE TITLE
More clarity around passed case 4 for Element with aria-hidden has no focusable content [6cfa84]

### ISSUE THREAD

**Issue by straker** [act-rules/act-rules.github.io#1811]
Axe-core recently tried to implement the [passed example 4](https://act-rules.github.io/rules/6cfa84#passed-example-4) for Element with aria-hidden has no focusable content. I was hoping for some clarity around the example, namely if you might have any real-world scenarios around the `onfocus` attribute.

Currently we don't have a way to determine what the `onfocus` attribute does on an element, so are left with just not failing any element with the attribute. This seems a bit too broad of a stroke though so were hoping to gather some more real-world examples of the `onfocus` attribute. I tried to find some real-world examples, but was unsuccessful (all I found were mostly just tutorials and explanations about the attribute).

**Comment by Jym77** [comment]
Hi @straker ,
would the case discussed in #643 answer your question? (it's not `onFocus`, but a similar use of redirecting focus with Javascript).
I do agree that static tools essentially cannot guess what Javascript is doing 😢

**Comment by Jym77** [comment]
@straker #1819 is improving the example to a more realistic one (elements behind a modal redirect focus while the modal is open).

**Comment by straker** [comment]
Yep, that's very helpful. Thanks for the updated example.
